### DISCUSSION TITLE
Push Access Denied

### DISCUSSION THREAD

**Discussion by EmeditWeb** [Tekipeps/aul-forum#62] [General]
I tried Push my Preloader work to the main branch so that bidemi can work on it and return the preloader to home page route but i was denied ,,,i also tried update my repo with the code and push it to the main branch, but got some errors,,what should i do?
## Errors Be like.
`PS C:\Users\emmanuel\Desktop\aul-forum> git push
remote: Permission to Tekipeps/aul-forum.git denied to EmeditWeb.
fatal: unable to access 'https://github.com/Tekipeps/aul-forum.git/': The requested URL returned error: 403
PS C:\Users\emmanuel\Desktop\aul-forum> git push 'https://github.com/Emeditweb/aul-forum.git/'
Enumerating objects: 111, done.
Counting objects: 100% (111/111), done.
Delta compression using up to 4 threads
Compressing objects: 100% (64/64), done.
Writing objects: 100% (93/93), 205.33 KiB | 4.28 MiB/s, done.
Total 93 (delta 27), reused 88 (delta 23), pack-reused 0
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date
PS C:\Users\emmanuel\Desktop\aul-forum>  git push 'https://github.com/Emeditweb/aul-forum.git/'
Enumerating objects: 111, done.
Counting objects: 100% (111/111), done.
Delta compression using up to 4 threads
Compressing objects: 100% (64/64), done.
Writing objects: 100% (93/93), 205.33 KiB | 4.03 MiB/s, done.
Total 93 (delta 27), reused 88 (delta 23), pack-reused 0
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date
PS C:\Users\emmanuel\Desktop\aul-forum> git push
remote: Permission to Tekipeps/aul-forum.git denied to EmeditWeb.
fatal: unable to access 'https://github.com/Tekipeps/aul-forum.git/': The requested URL returned error: 403`
     
Sorry for delays in my Contribution,i gat lot of works to do in Microbiology,,,you know this biology stuff na; 😌😌

**Comment by Tekipeps** [comment]
Go and watch the video on git i sent..
